app/components/primer/component.rb

```diff
@@ -7,5 +7,15 @@ class Component < ViewComponent::Base
     include FetchOrFallbackHelper
     include OcticonsHelper
     include JoinStyleArgumentsHelper
+
+    STATUSES = [
+      :experimental,
+      :beta,
+      :recommended,
+      :released,
+    ]
```

Comment:
nit: I actually prefer using a hash here


```rb
STATUSES = {
  experimental: :experimental,
  ...
```

and in the class

```rb
def self.status
  Primer::Component::STATUSES[:experimental]
end
```

Reply:
another idea would be transforming `status` into a DSL so we could add validations to ensure no invalid status are being used